py3-cms-tfaot.spec

```diff
@@ -0,0 +1,22 @@
+### RPM external py3-cms-tfaot 1.0.0
```

Comment:
@riga , for pip based packages it is batter to use the default way of building ( i.e. using https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_14_1_X/master/build-with-pip.file ). So please drop this spec and add `cmsdist/pip/cms-tfaot.file` with following contents
```
Requires: py3-cmsml py3-tensorflow
%define github_user riga
%define tag a2bbd06cbed0efa1fa191cc2f50a6b03067e59d2
%define branch master
%define source0 git+https://github.com/%{github_user}/cms-tfaot.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz 
```

Reply:
👍 Changed in 7838448.